src/site/_data/i18n/courses.yml

```diff
@@ -92,3 +92,9 @@ learn_html:
     en: Learn HTML!
   description:
     en: "This HTML course for web developers provides a solid overview for developers, from novice to expert level HTML."
+
+example:
```

Comment:
Did you verify this doesn't pop up anywhere where it should not? Algolia index, pages where courses are listed, ...?